xrootd/authorization_test.go

```diff
@@ -153,7 +169,7 @@ func TestWriteOriginScitokensConfig(t *testing.T) {
 	err = os.WriteFile(scitokensCfg, []byte(toMergeOutput), 0640)
 	require.NoError(t, err)
 
-	err = WriteOriginScitokensConfig()
+	err = WriteOriginScitokensConfig([]string{})
```

Comment:
Can you add a test which does a write with a non-empty array and test that the file generated contains what is expected?

Or am I missing seeing that being tested somewhere?

Reply:
Okay, with latest push both the generation of the `Issuer` struct and the file writing is tested.